sunpy/net/jsoc/tests/test_jsoc.py

```diff
@@ -80,17 +69,17 @@ def test_post_pass(client):
 def test_build_table(client):
     responses = client.search(
         a.Time('2020/1/1T00:00:00', '2020/1/1T00:00:45'),
-        a.jsoc.Series('hmi.M_45s'), a.jsoc.Notify('jsoc@cadair.com'))
+        a.jsoc.Series('hmi.M_45s'))
     table = responses.build_table()
     assert isinstance(table, astropy.table.Table)
 
     columns = ['T_REC', 'TELESCOP', 'INSTRUME', 'WAVELNTH', 'CAR_ROT']
-    assert columns == table.colnames
+    assert columns == table._display_table.colnames
```

Comment:
Is there no public way to get the column names from `table`?

Reply:
Yeah `.colnames` I think this is testing the behaviour of the private property on the base class.

Reply:
The `QueryResponseTable` allows some columns to be hidden when printing the table. This is done through the private `_display_table` property which is used in `__str__`, `__repr__` and `_repr_html_`. This test is testing the behaviour of that property.